central/compliance/checks/common/checks.go

```diff
@@ -212,66 +211,42 @@ This control checks that all deployments have ingress and egress network policie
 // Use this with DeploymentKind control checks.
 func CheckNetworkPoliciesByDeployment(ctx framework.ComplianceContext) {
 	// Map deployments to nodes.
-	networkGraph := ctx.Data().NetworkGraph()
-	deploymentIDToNodes := make(map[string]*v1.NetworkNode, len(networkGraph.GetNodes()))
-	for _, node := range networkGraph.GetNodes() {
-		if node.GetEntity().GetType() != storage.NetworkEntityInfo_DEPLOYMENT {
-			continue
-		}
-		deploymentIDToNodes[node.GetEntity().GetId()] = node
-	}
+	networkGraph := ctx.Data().DeploymentsToNetworkPolicies()
```

Comment:
[nit]: Maybe renaming this to `deploymentsToNetworkPolicies` since this is no longer a NetworkGraph. Also the comment above should be `Map deployments to NetworkPolicies`

Reply:
Will do